invenio_rdm_records/records/api.py

```diff
@@ -339,7 +341,7 @@ class RDMDraft(CommonFieldsMixin, Draft):
 
     model_cls = models.RDMDraftMetadata
 
-    index = IndexField("rdmrecords-drafts-draft-v6.0.0", search_alias="rdmrecords")
+    index = IndexField("rdmrecords-drafts-draft-v6.1.0", search_alias="rdmrecords")
```

Comment:
The mappings names need to align with the jsonschemas so we'll either need a new jsonschema or simply add it to existing v6.0.0. I just did a reindex (on the search cluster) of all of records/drafts zenodo today, so it's fairly ok - about 15min.

Reply:
:+1: Switched back to using 6.0.0, so it will require a re-indexing of Zenodo, but the mapping version will align exactly to the jsonschema version now.